homework/RosmaryFermin/Problem 2 Cards/src/nyc/c4q/RosmaryFC/cardGame.java

```diff
@@ -0,0 +1,11 @@
+package nyc.c4q.RosmaryFC;
+/**
+ * Created by c4q-rosmary on 5/2/15.
+ */
+public class cardGame
```

Comment:
a class cardGame should extend an abstract class Game. I cannot find a Game class